RFC-005-SWAP-Bitcoin-basic.md

```diff
@@ -0,0 +1,174 @@
+# Bitcoin Basic HTLC Atomic Swap
+
+- RFC-Number: 005
+- Status: Draft
+- Discussion-issue: -
+- Created on: 01 Mar. 2019
+
+# Table of contents
+
+<!-- toc -->
+
+- [Description](#description)
+- [Motivation](#motivation)
+- [Content](#content)
+  - [Identities](#identities)
+  - [Scripts](#scripts)
+    - [Variables](#variables)
+    - [HTLC](#htlc)
+      - [Human friendly - Opcode representation](#human-friendly---opcode-representation)
+      - [Implementation friendly - Hex representation](#implementation-friendly---hex-representation)
+    - [Redeem](#redeem)
+    - [Refund](#refund)
+    - [Timeouts](#timeouts)
+}- [Registry extension](#registry-extension)
+
+<!-- tocstop -->
+
+## Description
+
+This RFC defines the Bitcoin ledger and the Bitcoin asset for use in an execution [RFC002](./RFC-002-SWAP.md) SWAP with `comit-rfc-003` ([RFC003](./RFC-003-SWAP-basic.md)) as the SWAP protocol.
```

Comment:
:wrench: At first read, this sounds like you are repeating yourself from RFC4. Maybe start the sentence with what it actually defines?

```suggestion
This RFC defines how to execute an [RFC002](./RFC-002-SWAP.md) SWAP with `comit-rfc-003` ([RFC003](./RFC-003-SWAP-basic.md)) as the SWAP protocol for the Bitcoin ledger and the Bitcoin asset.
```

Reply:
Yeah I originally had this exact wording but I changed it because this RFC doesn't exclusively determine how to execute an RFC3 swap (rather both ledger's RFCs do). I will think some more about how to word this better.